### DISCUSSION TITLE
Students can't accept GH Classroom assignments if they aren't Members of the Organization

### DISCUSSION THREAD

**Discussion by KaterinaWalter** [community/community#139001] [GitHub Education]
### Hi everyone,

I recently transitioned to GitHub as the primary tool for my high school computer science classes. I’m running into an issue where I share an assignment link from GitHub Classroom, but several students are unable to accept it. I believe it’s related to their _membership status_ in the associated school Organization I created. 

**Bug Description:** I created a private template repository in my school organization to use as starter code for an assignment in GitHub Classroom. I sent the assignment link to 8 students, but 3 of them told me that they encountered the following error: 
> **"There was an error accepting the assignment. We were not able to add the user to your assignment, please try again.. Try and (update) the assignment. If this problem persists, please have the Classroom admin contact support."** 

**Additional Context:** All 8 students appear to be successfully “linked” on the Roster within the GitHub Classroom. HOWEVER – only the first 5 students were able to join the associated Organization as members. Those 5 students who are listed as Members were able to accept the assignment and generate a repository, while the 3 students who are NOT listed as Members in the Organization are having the issue described above. I also received an email about a failed invitation: "___ could not be invited because you don't have seats available". I was under the impression that an Organization with Education benefits would accommodate all of my students (I teach around 30-40 students total per year) and I would not have to pay for "seats."

**Solution Attempts:** 
1. I first tried inviting the remaining students to join the organization as members, but the option to send invitations is no longer available. 
2. Next, I tried someone’s solution from [this discussion](https://github.com/orgs/community/discussions/122130), where I created a commit (a trivial change in the README) on the repository that was generated from the assignment (not the template repository). _This did NOT work._ 
3. My next idea is to ensure ALL of my students apply for the Student Developer Pack – but this is tedious, so I don’t want to pursue this if it’s not relevant… 

Any solutions? Codespaces is the perfect IDE for our purposes, so I really want to continue using GitHub. But I need to be able to **generate private repositories for student projects** for the rest of the year. I have 26 other students (in my other classrooms) that I need to fix this for urgently!

**Comment by chrissphinx** [comment]
It seems there might've been a backend change that requires collaborators on repos to be in the member list for the organization, because previously when students were added to their repos they were listed as "outside collaborators" and they **did not** show up in the organization member list. Nor were they required to be added as an organization member, really unsure why this change was made or what the current status is @RyanHecht ???

**Reply by KaterinaWalter** [reply]
Agreed, it would make sense that a change like that may be the source of this issue. @RyanHecht on that note, just want to reiterate that I no longer have the option to send invitations to join my organization (BWL-CS), after the first few students who joined successfully. It now says "7 out of 1 seats filled" on my organization page, which may be relevant. If I could just invite all of my students to be Members I believe it would fix the assignment issue too.
